quarkus/service/src/test/java/org/apache/polaris/service/quarkus/catalog/IcebergCatalogTest.java

```diff
@@ -1843,6 +1845,55 @@ public void testConcurrencyConflictUpdateTableDuringFinalTransaction() {
         .hasMessageContaining("conflict_table");
   }
 
+  @ParameterizedTest
+  @ValueSource(booleans = {false, true})
+  public void testTableOperationsDoesNotRefreshAfterCommit(boolean updateMetadataOnCommit) {
+    if (this.requiresNamespaceCreate()) {
```

Comment:
This is always true for Polaris, is it not?

Reply:
Good point, changed to `Assumptions.assumeTrue`